music21/roman.py

```diff
@@ -2488,10 +2488,10 @@ def _parseFigure(self):
         if not isinstance(self._figure, str):  # pragma: no cover
             raise RomanNumeralException(f'got a non-string figure: {self._figure!r}')
 
-        if not self.useImpliedScale:
-            useScale = self._scale
-        else:
+        if self.useImpliedScale:
```

Comment:
perhaps a bit more elegant, but this change is not really in the scope of the PR.  :-)